cmd/crc/cmd/setup.go

```diff
@@ -58,7 +58,7 @@ func runSetup(arguments []string) error {
 	}
 
 	if !config.Get(crcConfig.Bundle).IsDefault {
-		if err := validation.ValidatePath(config.Get(crcConfig.Bundle).AsString()); err != nil {
+		if err := validation.ValidateBundlePath(config.Get(crcConfig.Bundle).AsString(), crcConfig.GetPreset(config)); err != nil {
```

Comment:
Before this change, there was no warning displayed at all? commit log is not clear about that.

Reply:
Answering  myself, yes, no warnings before this commit, so switching to `ValidateBundlePath` allows to show a warning when the specified bundle does not correspond to the expected one. Would be good to explicitly mention in the commit log as this explains what is being done, describes how to test it.